### DISCUSSION TITLE
General guidance regarding Discussions and Issues

### DISCUSSION THREAD

**Discussion by Taitava** [Taitava/obsidian-shellcommands#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

Some general principles:
- ~~If you encounter a bug and you are pretty certain it is a bug, you can still open a new issue directly.~~
- ~~If you have a feature request that is quite simple and that has clear benefits and probably does not require much discussion about how it should look like from the user's perspective, you can still open a new issue directly.~~
- ~~If you are unsure whether a problem you encountered is a bug or if it could be solved with better guidance/documentation, then open a new discussion. Of course for clear documentation errors (README.md or in application guidance) you can open an issue directly.~~
- ~~If you have a feature request, but you are unsure how it should finally work, or unsure if it's possible to create, or you feel that it might otherwise need extensive discussion before implementation, then open a new discussion.~~
- Discussions can always be transformed into issues, if we later decide that a discussion should lead to concrete development.
- ~~Also issues can be transformed into discussions.~~

Edit 2024-01-06: Please use discussions for everything: bug reports, feature requests and other messages. I will then create issues from discussions myself, if needed. 🙂 

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Taitava** [comment]
# Categories - where to post and what to post
## Announcements _(locked)_
 - Future coming, already decided big features, but only if they don't have a discussion or issue elsewhere.
 - Just general news.
 - Not very active.

## Debugging and testing
 - For mystery problems and suspected bugs.
 - For [SC test suite](https://github.com/Taitava/obsidian-shellcommands-test-suite) related stuff once it gets released to the public.

## Ideas
- Feature requests.
- Users should post here rather than creating new [issues](https://github.com/Taitava/obsidian-shellcommands/issues) directly. I will then create new issues after a discussion, if the feature will be implemented.

## Notes _(locked)_
General planning guidelines and other instructions mainly relevant for me.

## Offtopic
 - Anything **not related** to this plugin. Just come to say hi!
 - When you read other people's posts in this GitHub repository, you'll find out how they are using Obsidian, and might want to comment or ask tips from them - not only related to this plugin, but to Obsidian or other things in general.

## Q & A
Questions like:
 - _How do I do thing X with Shell commands?_ / _Can SC be used for thing X?_
 - General help related to Windows/Linux/Mac shell commands, even if you don't plan to necessarily use this plugin to execute them. I'm glad to help also when this plugin is not involved in the question. 🙂 

## Show and tell
Things that you have accomplished in Obsidian with this plugin, and how this plugin has affected your workflow. Get ideas and inspiration when you see what kind of things others have done! 🌞 

---

_(Locked)_: A category where only maintainers (currently just me) can create new discussions, but everybody can read and answer.

**Reply by LennartCode** [reply]
Hi :)
found a small typo in this comment: The "Locked" in the heading "Notes" misses a C :)

**Reply by Taitava** [reply]
Oh, nice. Thank you! I fixed it. 🙂
